README.md

```diff
@@ -77,9 +77,11 @@ process itself.
 
 | Name | Version |
 |------|---------|
-| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 1.1.0 |
+| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 1.1.0, < 1.3.0 |
```

Comment:
Adding in a constraint here, since the [aws-ia/vpc/aws v1.4.1](https://registry.terraform.io/modules/aws-ia/vpc/aws/1.4.1) module does not support `1.3.0`.